_dev/src/ts/types/apiTypes.ts

```diff
@@ -7,10 +8,12 @@ interface ApiResponseHydration {
 }
 
 interface ApiResponseNextRoute {
+  kind: 'next';
```

Comment:
```suggestion
  kind: 'next_route';
```
I prefer to use `next_route` instead of `next` because we use the key `next` inside `interface ApiResponseAction` and it can be a little confused. What do you think ?
But for me it's not a blocking request.